addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -0,0 +1,592 @@
+                parent_node.remove(node)
+
+        cleanup_node(None, tree)
+
+        return etree.tostring(tree, pretty_print=True, encoding='unicode')
+
+    # -------------------------------------------------------------------------
+    # ELECTRONIC ADDRESS SCHEME (EAS), see https://docs.peppol.eu/poacc/billing/3.0/codelist/eas/
+    # -------------------------------------------------------------------------
+
+    def _get_eas_mapping(self):
+        return COUNTRY_EAS
+
+    # -------------------------------------------------------------------------
+    # UNIT OF MEASURE
+    # -------------------------------------------------------------------------
+
+    def _get_uom_unece_mapping(self):
+        return UOM_TO_UNECE_CODE
+
+    def _get_uom_unece_code(self, line):
+        # list of codes: https://docs.peppol.eu/poacc/billing/3.0/codelist/UNECERec20/
+        # or https://unece.org/fileadmin/DAM/cefact/recommendations/bkup_htm/add2c.htm (sorted by letter)
+        xmlid = line.product_uom_id.get_external_id()
+        if xmlid and line.product_uom_id.id in xmlid and len(xmlid[line.product_uom_id.id].split('.')) == 2:
+            return UOM_TO_UNECE_CODE.get(xmlid[line.product_uom_id.id].split('.')[1], 'C62')
+        return 'C62'
+
+    # -------------------------------------------------------------------------
+    # TAXES
+    # -------------------------------------------------------------------------
+
+    def _get_tax_unece_codes(self, invoice, tax):
+        """
+        Source: doc of Peppol (but the CEF norm is also used by factur-x, yet not detailed)
+        https://docs.peppol.eu/poacc/billing/3.0/syntax/ubl-invoice/cac-TaxTotal/cac-TaxSubtotal/cac-TaxCategory/cbc-TaxExemptionReasonCode/
+        https://docs.peppol.eu/poacc/billing/3.0/codelist/vatex/
+        https://docs.peppol.eu/poacc/billing/3.0/codelist/UNCL5305/
+        :returns: (tax_category_code, tax_exemption_reason_code, tax_exemption_reason)
+        """
+        supplier = invoice.company_id.partner_id.commercial_partner_id
+        customer = invoice.commercial_partner_id
+
+        # add Norway, Iceland, Liechtenstein
+        european_economic_area = self.env.ref('base.europe').country_ids.mapped('code') + ['NO', 'IS', 'LI']
+
+        if customer.country_id.code == 'ES':
+            if customer.zip[:2] in ['35', '38']:  # Canary
+                # [BR-IG-10]-A VAT breakdown (BG-23) with VAT Category code (BT-118) "IGIC" shall not have a VAT
+                # exemption reason code (BT-121) or VAT exemption reason text (BT-120).
+                return 'L', None, None
+            if customer.zip[:2] in ['51', '52']:
+                return 'M', None, None  # Ceuta & Mellila
+
+        # see: https://anskaffelser.dev/postaward/g3/spec/current/billing-3.0/norway/#_value_added_tax_norwegian_mva
+        if customer.country_id.code == 'NO':
+            if tax.amount == 25:
+                return 'S', None, 'Output VAT, regular rate'
```

Comment:
```suggestion
                return 'S', None, _("Output VAT, regular rate")
```
/!\ translations!

Reply:
I thought this was not meant to be translated but ok :)